scipy/signal/_filter_design.py

```diff
@@ -1128,6 +1128,30 @@ def zpk2tf(z, p, k):
     a : ndarray
         Denominator polynomial coefficients.
 
+Examples
+    --------
+    Find the polynomial representation of a transfer function H(s)
+    using its 'zpk' (Zero-Pole-Gain) representation.
+    
+    .. math::
+            
+        H(z) = 5 \frac
+        { (s - 2)(s - 6) }
+        { (s - 1)(s - 8) }
+
+    >>> from scipy.signal import zpk2tf
+    >>> z   = [2,   6]
+    >>> p   = [1,   8]
+    >>> k   = 5
+    >>> zpk2tf(z, p, k)
+    (   array([  5.,    -40.,   60.]), 
+        array([  1.,    -9. ,   8. ]))
+    
+    .. math::
+            
+        H(z) = \frac
+        { (5s^2 - 40s + 60) }
+        { ( s^2 -  9s +  8) }
```

Comment:
Here and in the other examples I think it is final to leave to final result as the output of code rather than restating

Reply:
Sure, thanks!

Reply:
done.